javascript/ql/experimental/adaptivethreatmodeling/test/generic_feature_testing/TestUtil.qll

```diff
@@ -0,0 +1,17 @@
+import javascript
+import experimental.adaptivethreatmodeling.FeaturizationConfig
+
+/**
+ * A featurization config that featurizes all endpoints.
+ *
+ * Ideally this should not be in here, but it is needed for EnclosingFunctionName and EnclosingFunctionBody due to performance considerations :(.
+ */
+class NoRestrictionsFeaturizationConfig extends FeaturizationConfig {
+  NoRestrictionsFeaturizationConfig() { this = "NoRestrictionsFeaturization" }
+
+  override DataFlow::Node getAnEndpointToFeaturize() { any() }
+}
+
+class Endpoint extends DataFlow::Node {
+  Endpoint() { this.asExpr().(VarAccess).getName() = "endpoint" }
+}
```

Comment:
You should be able to `import experimental.adaptivethreatmodeling.modelbuilding.extraction.NoFeaturizationRestrictionsConfig`

Reply:
I thought so too, but I'm not able to resolve that import locally though, neither is the CI check.
Could this be another instance of ATM packaging shenanigans, or am I missing something simple?

Reply:
Apologies, I got the wrong import.  It should be `import extraction.NoFeaturizationRestrictionsConfig`.

Reply:
Right, that works. Slightly embarrassed thanks from me :).

Commit pushed.